src/OrleansRuntime/ReminderService/LocalReminderService.cs

```diff
@@ -201,17 +203,28 @@ public Task Stop()
         private async Task ReadAndUpdateReminders()
         {
             if (stoppedCancellationTokenSource.IsCancellationRequested) return;
+            localRangeSequence++;
+            var cachedLocalRangeSequence = localRangeSequence;
```

Comment:
if we go with that solution, record the cachedLocalTableSequence next to where the range changes.

BTW, the 2 sequence numbers are very confusing now. I got confused between them more than once now. Lets call the new one something else, rangeSerialNumber. and rangeSerialNumberLocalCopy. Or anythijng else, which has a different name from localTableSequence .